CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ Contributors must follow the guidelines below:
    separate commits addressing feedback is that it provides an easy way for
    reviewers to see only what changed if desired.
 
-# Copying
+## Copying
```

Comment:
Why did linting want this change?

Reply:
Multiple top-level headings in the same document: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document

Reply:
Oh, that's kind of a weird one